src/models/Settings.ts

```diff
@@ -108,6 +108,7 @@ const local: Record<string, ModelProperty> = {
   'features.showIdentifier': ['boolean', false, true],
   'features.autoFocus': ['boolean', false, true],
   'features.rememberMyUsernameOnOIE': ['boolean', false, false],
+  'features.showSessionRevocation': ['boolean', false, false],
```

Comment:
Should add to the classic readme under docs/

Reply:
thank you for catching that, I wasn't aware of it at all!